scriptworker/task.py

```diff
@@ -600,16 +617,16 @@ def prepare_to_run_task(context, claim_task):
         'env': env,
     }
 
-    subprocess = await asyncio.create_subprocess_exec(*context.config['task_script'], **kwargs)
-    context.proc = await to_cancellable_process(TaskProcess(subprocess))
-    timeout = context.config['task_max_timeout']
+    subprocess = await asyncio.create_subprocess_exec(*task_context.config['task_script'], **kwargs)
```

Comment:
You'll need to pass that workdir here, rather than having it be a part of every script's config, if you want to handle multiple tasks simultaneously.

Reply:
+1, that's on my list. For the monorepo scripts, I'll look at passing args in the commandline... perhaps multiple tasks won't be suppored without that? I also thought about a config template, but I don't love that.

Reply:
Also, looking at other bits, the artifact directory should be passed.

I think we can probably just require new args; the only script (I think) that doesn't use `scriptworker.client` or `scriptworker_client` is balrogscript, which hasn't been updated to since it was ported to py3.